.vscode/launch.json

```diff
@@ -12,10 +12,9 @@
                  "--language",
                  "typescript",
                  "-o",
-                 "${workspaceFolder}/samples/msgraph-mail/typescript/src",
```

Comment:
please revert the changes on launch.json for the output path and the working directory. We can keep the namespace change for PHP if you want

Reply:
right, that was to reduce the noise from dependabot. I still have the samples repo cloned under the samples directory under the kiota repo itself though (hence why it's excluded)
`gh repo clone microsoft/kiota-samples samples`

I had originally left the paths as is because I figured most people would be in the same case moving out of the submodule, we could change that assuming most devs on the team are in your case.

Reply:
(bottom line, could we handle that separately please?)